packages/ui/src/theme/platforms/react-native/transforms.ts

```diff
@@ -0,0 +1,102 @@
+import { DesignToken, Transform } from 'style-dictionary';
+import {
+  isBorderWidth,
+  isFontSize,
+  isFontWeight,
+  isOpacity,
+  isRadius,
+  isShadow,
+  isSpace,
+  isTime,
+} from './matchers';
+
+/*
+TODO: * remove transitionDuration - not supported in RN
+      * shadows
+      * inherit
+      * auto
+      * padding
+      * gap
+      * boxShadow
+      * remove any units (px, rem, em)
+*/
+
+export const ReactNativeTransforms: Record<string, Transform> = {
+  RNFontWeight: {
+    type: 'value',
+    matcher: isFontWeight,
+    transformer: (token: DesignToken) => {
+      const intValue = parseInt(token.value, 10);
```

Comment:
I see that some of these transforms use `parseInt`, `parseFloat`, and `Math.floor`. I'm wondering which of these values can actually be floats, or if we can just use `parseInt` for everything.

Reply:
This is based on the current values in the corresponding design tokens. For example font sizes are in rems and are floats: https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/fontSizes.ts